packages/vite-plugin-cloudflare/README.md

```diff
@@ -0,0 +1,465 @@
+# `@cloudflare/vite-plugin`
+
+[Intro](#intro) | [Quick start](#quick-start) | [Tutorial](#tutorial) | [API](#api) | [Worker environments](#worker-environments) | [Migrating from `wrangler dev`](#migrating-from-wrangler-dev)
```

Comment:
I'm thinking now that it's clearer if we refer to these as 'Cloudflare environments' rather than 'Worker environments'. Two reasons: firstly the environment variable used is `CLOUDFLARE_ENV`; secondly, once we start talking about 'Vite environments', the term `Worker environments` is confusing as each Vite environment is also a Worker and could therefore be considered a Worker environment. No perfect solution here but hopefully this is better (have changed all the other references below).

```suggestion
[Intro](#intro) | [Quick start](#quick-start) | [Tutorial](#tutorial) | [API](#api) | [Cloudflare environments](#cloudflare-environments) | [Migrating from `wrangler dev`](#migrating-from-wrangler-dev)
```

Reply:
Yes this makes a lot of sense.